src/solvers/smt/smt_bitcast.cpp

```diff
@@ -1,6 +1,25 @@
 #include <solvers/smt/smt_conv.h>
 #include <util/type_byte_size.h>
 
+template <typename Extract>
+static std::pair<size_t, expr2tc>
```

Comment:
What's wrong with a `std::pair`? I'm not calling type_byte_size because this information is already computed and not doing it for each node in the tree saves half of those calls.